GliaWidgets/Sources/RemoteConfiguration/RemoteConfiguration+SecureConversations.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+extension RemoteConfiguration {
+    struct SecureConversations: Codable {
```

Comment:
For others it's also done like this? Because it's quite big, and then adding confirmation and chat transcript later will mean that this struct will have, what, 50 properties at least.

Reply:
Gerson is making a good point. I think this should be broken into smaller objects, similar to what a CallStyle, or ChatStyle has done.

Reply:
RemoteConfiguration defines JSON structure and now it has 2 level of nesting. If we would like to make a more complex structure and add one more nesting level, we should introduce one more base object and consider how it affects integrators and how difficult it could be to manually create JSON.
However, if we would like to have separate structures, imho, better to create a separate structure like this on the root level, something like "secureConversationWelcomeScreen" and "secureConversationXxxxScreen".

Reply:
I find the suggestion of `secureConversationsWelcomeScreen` a good enough compromise.